openedx/core/djangoapps/course_live/tab.py

```diff
@@ -28,6 +31,18 @@ class CourseLiveTab(LtiCourseLaunchMixin, TabFragmentViewMixin, EnrolledTab):
         'instructor': 'Administrator',
     }
 
+    @property
+    def link_func(self):
+        legacy_link_func = super().link_func
+
+        def _link_func(course, reverse_func):
+            if VIEW_COURSE_LIVE_IN_APP_LEARNING_MFE.is_enabled():
```

Comment:
Any specific reason for added this behind a flag?

Reply:
I had discussion with Ahtisham bhai. He said to add it behind the flag.